azure-devops/projects/hub-pa-projects/hub-pa-api.tf

```diff
@@ -1,29 +1,37 @@
-locals {
-  repository_name           = "hub-pa-api"
-  repository_repo_id        = "pagopa/${local.repository_name}"
-  repository_branch_name    = "main"
-  repository_pipelines_path = ".devops"
-  build_path                = "\\${local.repository_name}"
+variable "hub-pa-api" {
```

Comment:
@pasqualedevita Please note that the project is based on two different repositories. One for backend microservices, which is he one configured here; and a different one for frontend, which is meant to be build into static files and deployed to blob storage + cdn. You can find it [here](https://github.com/pagopa/hub-pa-fe)

Reply:
yes, we need to create a terraform file for each repository

Reply:
@GiovanniMancini I think that azure devpos pipeline defined here has legacy implementation https://github.com/pagopa/hub-pa-fe/tree/main/.devops
U can take more updated pipeline here https://github.com/pagopa/cgn-onboarding-portal-frontend/tree/master/.devops